adaptive/learner/skopt_learner.py

```diff
@@ -26,18 +28,23 @@ class SKOptLearner(Optimizer, BaseLearner):
     def __init__(self, function, **kwargs):
         self.function = function
         self.pending_points = set()
-        self.data = {}
+        self.data = collections.OrderedDict()
```

Comment:
In Python >= 3.6 all dicts are ordered in principle, but it is good to make it explicit.